cpp/example_code/dynamodb/Readme.md

```diff
@@ -1,30 +1,58 @@
-- **create_table.cpp** - Create an Amazon DynamoDB table.
-- **create_table_composite_key.cpp** - Create an Amazon DynamoDB table that has a composite key.
-- **delete_item.cpp** - Delete an item from an Amazon DynamoDB table.
-- **delete_table.cpp** - Delete an Amazon DynamoDB table.
-- **describe_table.cpp** - Retrieve information about an Amazon DynamoDB table.
-- **get_item.cpp** - Retrieve an item from an Amazon DynamoDB table.
-- **list_tables.cpp** - List all Amazon DynamoDB tables.
-- **put_item.cpp** - Place an item into an Amazon DynamoDB table.
-- **query_items.cpp** - Query an Amazon DynamoDB table.
-- **scan_table.cpp** - Scan an Amazon DynamoDB table.
-- **update_item.cpp** - Update an item in an Amazon DynamoDB table.
-- **update_table.cpp** - Update information about an Amazon DynamoDB table.
-
-To run these examples, you can setup your development environment. For more information, 
-see [Getting started using the AWS SDK for C++](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html). 
+# Amazon DynamoDb code examples for the SDK for C++ using the DynamoDBClient
+## Overview
+Sample code which demonstrates creation, deletion, modification, and querying of DynamoDB databases.
+
+*Amazon DynamoDB is a fully managed NoSQL database service that provides fast and
+predictable performance with seamless scalability.*
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+## Code examples
+### Single actions
+Code excerpts that show you how to call individual service functions.
+* [batch_get_item](./batch_get_item.cpp) (BatchGetItem)
+* [create_table](./create_table.cpp) (CreateTable)
+* [create_table_composite_key](./create_table_composite_key.cpp) (CreateTable)
+* [delete_item](./delete_item.cpp) (DeleteItem)
+* [delete_table](./delete_table.cpp) (DeleteTable)
+* [describe_table](./describe_table.cpp) (DescribeTable)
+* [get_item](./get_item.cpp) (GetItem)
+* [list_tables](./list_tables.cpp) (ListTables)
+* [put_item](./put_item.cpp) (PutItem)
+* [query_items](./query_items.cpp) (Query)
+* [scan_table](./scan_table.cpp) (Scan)
+* [update_item](./update_item.cpp) (UpdateItem)
+* [update_table](./update_table.cpp) (UpdateTable)
+### Scenarios
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+* [dynamodb_getting_started_scenario](./dynamodb_getting_started_scenario.cpp) (CreateTable, PutItem, UpdateItem, BatchWriteItem, GetItem, Query, Scan, DeleteItem, DeleteTable)
```

Comment:
Same thought about link title, perhaps this could be the same as the title of the scenario.